Framework/PythonInterface/mantid/plots/plotfunctions.py

```diff
@@ -378,8 +378,18 @@ def pcolormesh(axes, workspace, *args, **kwargs):
     return axes.pcolormesh(x, y, z, *args, **kwargs)
 
 
+def _skimage_version():
```

Comment:
You can use `LooseVersionCheck` from `distutils` ([example](https://github.com/mantidproject/mantid/blob/76178489f6bafb8165682400f5c269f96516f4df/Framework/PythonInterface/test/python/plugins/algorithms/StringToPngTest.py#L25))